shared/protocol/src/test/java/io/pravega/shared/protocol/netty/DelegatingRequestProcessorTest.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.shared.protocol.netty;
+
+import io.pravega.shared.protocol.netty.WireCommands.Event;
+import io.netty.buffer.Unpooled;
+import org.junit.Test;
+
+public class DelegatingRequestProcessorTest {
+
+    private static final Event EMPTY_EVENT = new WireCommands.Event(Unpooled.wrappedBuffer(new byte[1]));
+
+    private MockRequestProcessor rp = new MockRequestProcessor();
+
+    @Test
+    public void testEverythingCalled() {
```

Comment:
This doesn't actually assert anything. 
It should include an assertion that each method on the processor delegated to was invoked .